### PR TITLE
Use PostgreSQL 10

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,5 +3,5 @@ ENV LANG=en_US.UTF-8
 USER 0
 RUN yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
```
Comment (fridex) [inline Dockerfile:4]: The URL seems to be same based on https://www.postgresql.org/download/linux/redhat/

Comment (harshad16) [reply Dockerfile:4]: I believe its the same too. :)

**Comment by harshad16** [comment]
> This change depends on a change that failed to merge.

I don't know why this broke!

**Comment by fridex** [comment]
> > This change depends on a change that failed to merge.
> 
> I don't know why this broke!

I added `Depends-On` for Zuul so that it's not merged.

**Comment by harshad16** [comment]
recheck

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
@harshad16 PTAL, Zuul seems to be happy now :+1:

**Review by harshad16 — APPROVED** [review]

**Pull request merged**